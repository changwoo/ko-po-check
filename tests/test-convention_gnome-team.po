# -*-text-*-
# obsolete Korean L10N team address
msgid ""
msgstr ""
"Project-Id-Version: test 14\n"
"POT-Creation-Date: 2010-03-03 00:16+0900\n"
"PO-Revision-Date: 2010-03-03 00:16+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: GNOME Korea <gnome-kr-hackers@lists.kldp.net>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
