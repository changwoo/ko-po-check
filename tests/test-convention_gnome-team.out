-:4: 새 번역팀 주소를 사용하십시오: <gnome-kr@googlegroups.com>
